### ISSUE TITLE
[Bug] Filter+ deterioration

### ISSUE THREAD

**Issue by r2rstep** [darkreader/darkreader#11541]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this bug to avoid opening a duplicate.
- [X] I can reproduce this bug in a new, unmodified web browser profile with Dark Reader installed as the only extension.
- [X] I understand I need to use the [Broken Website Report template](https://github.com/darkreader/darkreader/issues/new?assignees=&labels=Broken+Website&template=broken-website-report.yml&title=%5BBroken+Website%5D+Replace+with+title) if this bug I am reporting occurs on a single website.

### Bug Description

I use Dark Reader on my desktop Brave browser. I've been using "Filter+" option for a while but since last week it doesn't work correctly on some sites such as stackoverflow.com, probasket.pl - the websites display correctly for half a second but then they go all white.

When I change the dark mode generation method to any other it works correctly.

I also checked when all other extensions are off and Brave shield is off but that doesn't help.

### Website Address

stackoverflow.com

### Steps To Reproduce

1. Set "Filter+" method
2. Go to stackoverflow.com

### Expected Behavior

correct dark theme generation

### Actual Behavior

websites are all white

### Operating System

Endeavour OS

### Web Browser name and version

Brave 1.56.11 based on Chromium 115.0.5790.102

### Dark Reader version

4.9.64

**Comment by KaKi87** [comment]
I'm also experiencing issues on calendar.google.com and deepl.com : pages don't get darkified at startup, but toggling Dark Reader off and on again, or refreshing, do turn the pages dark.

Thanks

**Comment by jettero** [comment]
I've been having problems with filter+ in firefox on nearly all the sites I used to use it. Mostly, I just switch those sites to filter (not plus) and it's acceptable, but I sure miss the old behavior.

I think pagerduty.com is the best example lately.

... the fact that more people aren't complaining about this leads me to wonder if it's something about my exotic setup that's the problem. If it is, I can't figure out what it would be.

**Comment by KaKi87** [comment]
Hello,

@Gusted, @alexanderby, @bershanskiy,

Any news on this ?

Thanks

**Comment by alexanderby** [comment]
Have you tried Dynamic or Filter mode?

**Comment by KaKi87** [comment]
Actually I am using *Dynamic*, and didn't realize *Filter+* was a different thing, sorry :sweat_smile: 

But, the fact is I'm experiencing the same issue OP mentioned :

> the websites display correctly for half a second but then they go all white

Thanks

**Comment by alexanderby** [comment]
There is a Filter+ bug in Firefox browser itself, it seems to be possible to make a quick fix. But I don't see any issues with Chromium. Do you have dark theme detection enabled? Maybe it detects a dark theme by mistake.

**Comment by KaKi87** [comment]
> Do you have dark theme detection enabled?

Nope, that has indeed been an issue before so I disabled a while ago.

Thanks
